cmd/vcert/validators.go

```diff
@@ -468,3 +496,44 @@ func validateRevokeFlags1(commandName string) error {
 
 	return nil
 }
+
+func validateOverWritingEnviromentVariables() {
+
+	colorYellow := "\033[33m"
+	colorReset := "\033[0m"
+
+	if getPropertyFromEnvironment(vCertURL) != "" {
+		if flags.url != "" {
+			logger.Println(colorYellow, "Warning Command line parameter -u has overridden environment variable VCERT_URL", colorReset)
+		}
+
+	}
+
+	if getPropertyFromEnvironment(vCertZone) != "" {
+		if flags.zone != "" {
+			logger.Println(colorYellow, "Warning: Command line parameter -z has overridden environment variable VCERT_ZONE", colorReset)
+		}
+
+	}
+
+	if getPropertyFromEnvironment(vCertToken) != "" {
+		if flags.tppToken != "" {
+			logger.Println(colorYellow, "Warning: Command line parameter -t has overridden environment variable VCERT_TOKEN", colorReset)
+		}
+	}
+
+	if getPropertyFromEnvironment(vCertApiKey) != "" {
+		if flags.apiKey != "" {
+			logger.Println(colorYellow, "Warning: Command line parameter -k has overridden environment variable VCERT_APIKEY", colorReset)
+		}
+
+	}
+
+	if getPropertyFromEnvironment(vCertTrustBundle) != "" {
+		if flags.trustBundle != "" {
+			logger.Println(colorYellow, "Warning: Command line parameter -trust-bundle has overridden environment variable VCERT_TRUST_BUNDLE", colorReset)
```

Comment:
Although both `-trust-bundle` and `--trust-bundle` are valid, we should use the `--trust-bundle` form in messages since that is meant to be the standard for CLI parameters consisting of more than one character.

Reply:
fixed on the last commit.